index.html

```diff
@@ -9,8 +9,11 @@
   <body>
     <div id="app">
     </div>
-    <script src="https://unpkg.com/jshint@2.9.6/dist/jshint.js"></script>
+    <p>
+      The datasim-ui project has been deprecated and this hosted webpage has been taken down. To continue using DATASIM, you can use the application <a href="https://github.com/yetanalytics/datasim#cli">directly from the CLI</a> or sign up for <a href="https://beta.centriph.com/">Centriph</a> to use its DATASIM features.
+    </p>
+    <!--script src="https://unpkg.com/jshint@2.9.6/dist/jshint.js"></script>
     <script src="https://unpkg.com/jsonlint@1.6.3/web/jsonlint.js"></script>
-    <script src="cljs-out/prod-main.js" type="text/javascript"></script>
+    <script src="cljs-out/prod-main.js" type="text/javascript"></script-->
```

Comment:
what's `--`?

Reply:
I commented out the three lines of `script` tags